torch/jit/__init__.py

```diff
@@ -1711,7 +1686,13 @@ def __getattr__(self, attr):
                 if original_module and self.__dict__["_initialized"]:
                     # get attr from original if it is still alive
                     return getattr(original_module, attr)
-
+                elif self.__dict__["_initialized"]:
+                    # original module is dead, try looking up the value on the
+                    # original type
+                    fn = getattr(self.__dict__["_original_type"], attr, None)
+                    if fn is not None and inspect.isfunction(fn):
+                        # bind the function to this instance and return it
+                        return fn.__get__(self, self.__dict__["_original_type"])
```

Comment:
What is causing this to be needed? We eventually cannot allow any of these fallbacks to happen.

Reply:
If a function is ignored then the resulting ScriptModule doesn’t know about it, so we need this so the Python function can be looked up.

Eventually we should just copy the ignored Python functions over all over when creating a ScriptModule which would let us delete this